db/seeds.rb

```diff
@@ -206,7 +206,7 @@
   title: "Welcome Notification",
   processed_html: "Welcome to dev.to! Start by introducing yourself in <a href='/welcome' data-no-instant>the welcome thread</a>.",
   type_of: "Onboarding",
-  sent: true,
```

Comment:
grepping for `sent` must have been fun :D

Reply:
LOL, it wasn't too terrible, but also not easy, either 😂